src/VaultSharp/VaultSharp.csproj

```diff
@@ -35,6 +35,12 @@
     <WarningLevel>4</WarningLevel>
     <DocumentationFile>bin\Release\VaultSharp.XML</DocumentationFile>
   </PropertyGroup>
+  <PropertyGroup>
+    <SignAssembly>true</SignAssembly>
+  </PropertyGroup>
+  <PropertyGroup>
+    <AssemblyOriginatorKeyFile>VaultSharp.snk</AssemblyOriginatorKeyFile>
```

Comment:
maybe i can hold off on a strong named nuget package. 
i am assuming this Pull Request, when merged, will solve your issue. 

i can evaluate others' needs on a case by case basis. the world seems to be divided when it comes to strong naming open source projects in NuGet.